autoblocks/_impl/testing/run.py

```diff
@@ -266,8 +265,6 @@ async def async_run_test_suite(
             evaluator_id=None,
             error=err,
         )
-    finally:
-        current_external_test_id.reset(token)
```

Comment:
you don't wanna do this anymore?

Reply:
this happens in the test case run itself now since I combined them.

Reply:
oh ya i see